src/main/java/com/conveyal/datatools/manager/jobs/FeedUpdater.java

```diff
@@ -91,36 +95,31 @@ public void run() {
             LOG.info("Running initial check for feeds on S3.");
             eTagForFeed = new HashMap<>();
         }
+        LOG.info("Checking for feeds on S3.");
         Map<String, String> newTags = new HashMap<>();
-        // iterate over feeds in download_prefix folder and register to gtfsApi (MTC project)
+        // iterate over feeds in download_prefix folder and register to (MTC project)
         ObjectListing gtfsList = FeedStore.s3Client.listObjects(feedBucket, bucketFolder);
+        LOG.info(eTagForFeed.toString());
         for (S3ObjectSummary objSummary : gtfsList.getObjectSummaries()) {
 
             String eTag = objSummary.getETag();
+            String keyName = objSummary.getKey();
+            LOG.info("{} etag = {}", keyName, eTag);
             if (!eTagForFeed.containsValue(eTag)) {
-                String keyName = objSummary.getKey();
                 // Don't add object if it is a dir
-                if (keyName.equals(bucketFolder)) {
-                    continue;
-                }
+                if (keyName.equals(bucketFolder)) continue;
                 String filename = keyName.split("/")[1];
                 String feedId = filename.replace(".zip", "");
                 // Skip object if the filename is null
                 if ("null".equals(feedId)) continue;
                 try {
-                    LOG.warn("New version found for at {}/{}. ETag = {}. Downloading from s3", feedBucket, keyName, eTag);
-                    S3Object object = FeedStore.s3Client.getObject(feedBucket, keyName);
-                    InputStream in = object.getObjectContent();
-                    File file = new File(FeedStore.basePath, filename);
-                    OutputStream out = new FileOutputStream(file);
-                    ByteStreams.copy(in, out);
-                    String md5 = HashUtils.hashFile(file);
+                    LOG.info("New version found for {} at s3://{}/{}. ETag = {}.", feedId, feedBucket, keyName, eTag);
                     FeedSource feedSource = null;
                     List<ExternalFeedSourceProperty> properties = Persistence.externalFeedSourceProperties.getFiltered(and(eq("value", feedId), eq("name", AGENCY_ID)));
```

Comment:
Please make this expand to a few lines to be more readable.  Also, see https://github.com/conveyal/datatools-server/pull/153/files#r245153439.